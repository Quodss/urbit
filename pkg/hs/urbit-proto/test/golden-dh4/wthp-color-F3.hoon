%.  %b
|=  color=?($r $g $b)
^-  @t
?-  color
  %r  'red'
==
