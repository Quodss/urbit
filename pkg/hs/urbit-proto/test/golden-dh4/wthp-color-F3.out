PARSED

%.  %b
|=  color=?(%r %g %b)
^-  @t
?-(color %r "red")

SOFT

%-  |=  color=?(%b %g %r)
    ^-  @t
    ??  ?=(%114 color)
    "red"
%b

ERROR

  trace:  play:
          lvl   0
          sut   0|*
          ----
          code  %-  |=  color=?(%b %g %r)
                    ^-  @t
                    ??  ?=(%114 color)
                    "red"
                %b


          play:
          lvl   0
          sut   0|*
          ----
          code  |=  color=?(%b %g %r)
                ^-  @t
                ??  ?=(%114 color)
                "red"


          tire:
          fish
                >>  [%114 _]


          type  [color=?(%b %g %r) 0|*]


  tire-fish:
  axis  2
  miss  [%98 _]

